innopoints/views/project.py

```diff
@@ -74,6 +62,14 @@ def list_projects():
         if order_by == 'proximity':
             db_query = db_query.join(Activity).group_by(Project.id)
         db_query = db_query.order_by(ordering[order_by, order])
+    elif request.args.get('type') == 'past':
+        db_query = db_query.filter(or_(Project.lifetime_stage == LifetimeStage.finalizing,
+                                       Project.lifetime_stage == LifetimeStage.finished))
+        page = int(request.args.get('page', 1))
+        db_query = db_query.order_by(Project.id.desc())
```

Comment:
`order_by` cannot be used with `type=past`?

Reply:
No, there's no ordering in the UI. Not even the search, though this will be added someday

Reply:
Then let's add an issue to the backlog milestone